src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/ActivityScheduleService.kt

```
@@ -104,7 +104,7 @@ class ActivityScheduleService(
 
     schedule.allocatePrisoner(
       prisonerNumber = prisonerNumber,
-      bookingId = prisonerDetails.bookingId,
+      bookingId = prisonerDetails.bookingId ?: throw IllegalStateException("Active prisoner $prisonerNumber does not have a booking id."),
```

Should never be null.  Fail fast if it is.